### ISSUE TITLE
Training loss

### ISSUE THREAD

**Issue by ashwanijha1** [SamsungSAILMontreal/ForestDiffusion#13]
Hi,

I just wanted to know if it is possible to extract a training curve (loss by learning step) or something similar from the model please? 

I'm working with the python-based model on Linux Mint.

Thanks,
Ash

**Comment by AlexiaJM** [comment]
You would need to put "verbose=True" or "verbose=2" in https://github.com/SamsungSAILMontreal/ForestDiffusion/blob/818ac3b9c8df7b7c763470c9a0691dacca8d8b37/Python-Package/base-ForestDiffusion/ForestDiffusion/diffusion_with_trees_class.py#L246C3-L246C80. This gives you loss wrt the number of trees. I'm not sure that it would work with n_jobs != 1. I wish that there was a better way. 

You can also find the XGBoost boosters by doing your_model.regr[i][j], but I don't think that you can get much from these.

**Comment by ashwanijha1** [comment]
Thanks so much.  I suppose I was thinking of ways to try to compare/de-bug the model fit across models. I'll start a new thread about it maybe...
